### DISCUSSION TITLE
anyone have proxxies which works

### DISCUSSION THREAD

**Discussion by pl4udible** [3kh0/ext-remover#301] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
most proxxies are blocked

give me one that works and is not blocked with Lightspeed filter agent

**Comment by rwg100** [comment, marked as answer]
tadaaaaa idk what the hell lightspeed filter is

[https://bananaboat.me/](url)

[https://womginx1.lunarvcoding.repl.co/](url)

**Reply by pl4udible** [reply]
Lightspeed filter agent

**Reply by pl4udible** [reply]
and it does not work, when  i go on youtube it shows a blank page

**Reply by Aka-but-nice** [reply]
Here's a proxy to keep yall occupied https://aadynpersonalunblocker.glitch.me/ if yall want Roblox go to this link in the proxy https://kxd.fm/roblox/ YOU HAVE TO USE THE PROXY FOR IT, https://kxd.fm/roblox/ will not work if you use it in your regular browser! Also check my site it has proxies and games and other stuff, best-site-everr.glitch.me

**Reply by rwg100** [reply]
oh yeah it wasnt made to work with youtube. its slow on a lot of websites.

**Comment by nicolasog12** [comment]
i gotchu
https://all-star-math.ml
https://all-star-ela.ml

**Comment by PickleIsDaBest** [comment]
https://pineappleeater.glitch.me/
